python/lsst/dax/apdb/cassandra/apdbCassandraReplica.py

```diff
@@ -84,6 +84,10 @@ def _timer(self, name: str, *, tags: Mapping[str, str | int] | None = None) -> T
         """Create `Timer` instance given its name."""
         return Timer(name, _MON, tags=tags)
 
+    def schemaVersion(self) -> VersionTuple:
+        # Docstring inherited from base class.
+        return self._schema.schemaVersion()
```

Comment:
This is not the version you are looking for 🙂 This version is what defined in felis, to get database schema version you need:

    return self._apdb._db_versions.schema_version